governance/polls/Proposal - Increase System Surplus FLOP Threshold

```diff
@@ -0,0 +1,37 @@
+---
+title: Increase the System Surplus FLOP Threshold to 2 Million Dai - August 10, 2020
```

Comment:
I think you may need to clarify the name of what we're changing. It doesn't directly involve FLOP functionality. See https://docs.makerdao.com/smart-contract-modules/system-stabilizer-module/vow-detailed-documentation.

I'd suggest 'Surplus Auction Buffer'

Reply:
Oh, and can you make sure the filename as the .md suffix, please.

Reply:
Maybe use DAI instead of Dai. Super minor point though, consistency isn't that important.